calendar-helper/index.js

```diff
@@ -88,15 +121,31 @@ class CalendarHelper {
 			//currentRow[weekDay] = CalendarHelper.toYyyymmdd(date);
 			if (d < minDay || d > maxDay) {
 				currentRow[weekDay] = m.callbackButton(CalendarHelper.strikethroughText(d.toString()), "calendar-telegram-ignore-" + CalendarHelper.toYyyymmdd(date));
```

Comment:
IMO it would be better to never display the number with a strikethrough in this case.
```suggestion
				if (!this.options.hideIgnoredWeeks) {
					currentRow[weekDay] = m.callbackButton(CalendarHelper.strikethroughText(d.toString()), "calendar-telegram-ignore-" + CalendarHelper.toYyyymmdd(date));
				}
```